### ISSUE TITLE
Link text as new suspended tab title

### ISSUE THREAD

**Issue by QuentinCaffeino** [deanoemcke/thegreatsuspender#598]
Good day,

Is it possible to make tabs opened from right click (Open link in new suspended tab) take text from link as a title?
Why? Because when I'm reading some article and it has a lot of links to other articles. So as a result I have a dozen of tabs with same name.

Best

**Comment by deanoemcke** [comment]
This feature has actually been rewritten in the upcoming release (v7.x).
It will now actually load the page and then suspend it. This gives it access to the real page title and favicon.
See https://github.com/deanoemcke/thegreatsuspender/issues/572

**Comment by baz357** [comment]
23 Oct 2017 was many moons ago... can you give us a hint when v7.x will be released? :)

**Comment by deanoemcke** [comment]
Follow this issue for latest updates: https://github.com/deanoemcke/thegreatsuspender/issues/537
tldr: really soon!
